pennylane/ftqc/conditional_measure.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Contains the condition transform.
+"""
+from functools import wraps
+from typing import Callable
+
+import pennylane as qml
+from pennylane.measurements import MeasurementValue, MidMeasureMP
+from pennylane.ops.op_math.condition import CondCallable, Conditional
+
+
+def cond_meas(
+    condition,
+    true_fn: Callable,
+    false_fn: Callable,
+):
+    """Conditions the basis of mid-circuit qubit measurements on parameters such as the results of
+    other mid-circuit qubit measurements.
+
+    .. note::
+
+        This function is currently not compatible with :func:`~.qjit`, or with
+        :func:`.pennylane.capture.enabled`.
+
+    Args:
+        condition (Union[.MeasurementValue, bool]): a conditional expression that may involve a mid-circuit
+           measurement value (see :func:`.pennylane.measure`).
+        true_fn (callable): The quantum function or PennyLane operation to
+            apply if ``condition`` is ``True``. The callable must create a single mid-circuit measurement.
+        false_fn (callable): The quantum function or PennyLane operation to
+            apply if ``condition`` is ``False``. The callable must create a single mid-circuit measurement.
+
+    .. note::
+        The mid-circuit measurements applied the two branches must both be applied to the same
```

Comment:
[Grammar] I assume this is what you meant?

```suggestion
        The mid-circuit measurements applied on the two branches must both be applied to the same
```